### PR TITLE
Remove {highway: type} fallback from bike / walk profile, fixes #3231

### LINKED ISSUE #3231: Remove {highway: type} fallback from default bike / walk profile
See https://github.com/Project-OSRM/osrm-text-instructions/issues/62: there is no reason to ever return an encoding not intended for humans in the response. We removed it from the car profile long ago, and although at the moment we focus on car routing, bike and walk should return reasonable responses.

- [x] remove https://github.com/Project-OSRM/osrm-backend/blob/4ba8ccfcce50c1d1c6e91df34b85ecc98a116ec2/profiles/foot.lua#L157
- [x] remove https://github.com/Project-OSRM/osrm-backend/blob/4ba8ccfcce50c1d1c6e91df34b85ecc98a116ec2/profiles/bicycle.lua#L221

### PR DESCRIPTION
**Pull request by daniel-j-h**
For #3231: there should be no reason our _default_ profiles expose internals to the user.

## Tasklist
 - [x] review
 - [x] adjust for comments

References:
- https://github.com/Project-OSRM/osrm-text-instructions/issues/62#issuecomment-257889721
- https://github.com/Project-OSRM/osrm-text-instructions/pull/63

### CODE REVIEW

**Comment by TheMarex** [comment]
Looks good to me. :+1:

File: features/foot/ref.feature [hunk 1]
```diff
@@ -29,8 +29,8 @@ Feature: Foot - Way ref
             | ab    |      | E7  |
 
         When I route I should get
-            | from | to | route                               |
-            | a    | b  | {highway:primary},{highway:primary} |
+            | from | to | route |
```
Comment (MoKob) [inline features/foot/ref.feature]: shouldn't this test (since it has refs) check for refs?

Comment (daniel-j-h) [inline features/foot/ref.feature]: :+1:

**Comment by daniel-j-h** [comment]
This was merged into master and the branch then deleted.
Seems like Github saw this as first deleting the branch and then merging.

**Pull request closed without merging**